helm/oauth2-proxy/values.yaml

```diff
@@ -29,8 +29,8 @@ config:
   # existingConfig: config
 
 image:
-  repository: "quay.io/pusher/oauth2_proxy"
-  tag: "v5.1.0"
+  repository: "quay.io/oauth2-proxy/oauth2_proxy"
```

Comment:
Wrong URL. The right one is `quay.io/oauth2-proxy/oauth2-proxy`
`-` instead of `_`